src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,14 @@
+class PokemonClient {
+	constructor() {}
```

Comment:
The indentation on all files makes the code a bit hard to read, what tab space are you using? consider using 2, it will make the code a lot more readable